Code/Source/VehicleDynamics/DriveModels/SimplifiedDriveModel.h

```diff
@@ -39,5 +39,10 @@ namespace VehicleDynamics
         AZStd::vector<SteeringDynamicsData> m_steeringData;
         PidConfiguration m_steeringPid;
         PidConfiguration m_speedPid;
+
+        float maxSpeedImpulse = 0.0f;
```

Comment:
This does not look like a good default. The component in starting state would prevent vehicle from moving. 
Set this to whatever makes sense for a typical vehicle as a maximum. This should not prevent things working out of the box unless someone has an uncommon use case.

Reply:
I understand that 0 means disable.

Reply:
If so, also add a comment next to this default.

Reply:
Would no limit provide a better experience for a user as a default than a limit set to some value, say 500?

Reply:
Added comment in the code.

You're right, when zero is set, the limit is disabled (it's mentioned in the pop-up message). I'm not sure, what is the best value, even for the kraken (not mentioning any general formula). However, when I think about it, the reasonable starting (default) value would be great, especially for our demo. And this applies to all settings around (mainly PIDs). The best, general solution would be to calculate these values automagically, but I'm not sure we have time/resources to develop this (and - what's probably more time-consuming - to test this) right now. But that's a topic for a different discussion.